inrupt/pod-browser
# Using full VC in consent request page

# Use full VC in query params

This is so that we can make request between pods. 

To test: 

1 - logged into pod A (requestor), go to privacy > generate request
2 - add pod B webId as requestor URL and request a resource in pod B
3 - click generate request and copy THE FIRST URL you're redirected to before the authentication flow kicks you out of the consent flow. Note: it might be the auth flow lets you see the consent request properly, but still make sure the resource owner can access it properly.
4 - log in a resource owner (pod b) and paste the URL you copied before
5 - verify the consent request is displayed as expected and that you can accept it

# Checklist
